java/src/main/native/src/TableJni.cpp

```diff
@@ -3577,7 +3574,7 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_sample(JNIEnv *env, jclas
   JNI_NULL_CHECK(env, j_input, "input table is null", 0);
   try {
     cudf::jni::auto_set_device(env);
-    cudf::table_view *input = reinterpret_cast<cudf::table_view *>(j_input);
+    auto input = reinterpret_cast<cudf::table_view *>(j_input);
```

Comment:
```suggestion
    auto input = reinterpret_cast<cudf::table_view const *>(j_input);
```

Reply:
Fixed this occurrence, and other locations modified in this PR.